src/test/java/org/junit/rules/RuleChainTest.java

```diff
@@ -55,4 +63,41 @@ public void executeRulesInCorrectOrder() throws Exception {
                 "finished outer rule");
         assertEquals(expectedLog, LOG);
     }
+
+    @Test
+    public void aroundShouldNotAllowNullRules() {
+        RuleChain chain = RuleChain.emptyRuleChain();
+        try {
+            chain.around(null);
+            fail("around() should not allow null rules");
+        } catch (NullPointerException e) {
+            assertThat(e.getMessage(), equalTo("The enclosed rule should not be null"));
+        }
+    }
+
+    public static class RuleChainWithNullRules {
+        @Rule
+        public final RuleChain chain = outerRule(new LoggingRule("outer rule"))
+                .around(null);
+
+        @Test
+        public void example() {}
+    }
+
+    @Test
+    public void whenRuleChainHasNullRuleTheStacktraceShouldPointToIt() {
+        Result result = JUnitCore.runClasses(RuleChainWithNullRules.class);
+
+        assertThat(result.getFailures().size(), equalTo(1));
+        String stacktrace = stacktraceToString(result.getFailures().get(0).getException());
+        assertThat(stacktrace, containsString("\tat org.junit.rules.RuleChainTest$RuleChainWithNullRules.<init>(RuleChainTest.java:"));
+    }
+
+    // TODO delete as soon as #1312 is merged
+    private static String stacktraceToString(Throwable e) {
```

Comment:
In case you are going to merge #1312 soon, please merge that before this, so that I can remove this method.